File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/SedJournalføringMigreringRapportDto.java
package no.nav.melosys.eessi.service.sed;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class SedJournalføringMigreringRapportDto {
    private final List<SedMottattMigreringRapportDto> sedMottattMigreringRapportDtoList;
    private final List<SedSendtMigreringRapportDto> sedSendtMigreringRapportDtoList;
    private int antallSedMottattHendelser;
    private int antallSedSjekket;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/SedJournalføringMigreringRapportDto.kt
package no.nav.melosys.eessi.service.sed

data class SedJournalføringMigreringRapportDto(
    var sedMottattMigreringRapportDtoList: MutableList<SedMottattMigreringRapportDto?>? = null,
    var sedSendtMigreringRapportDtoList: MutableList<SedSendtMigreringRapportDto?>? = null,
    var antallSedMottattHendelser: Int = 0,
    var antallSedSjekket: Int = 0
)
